connector-setup/index.js

```diff
@@ -128,6 +128,17 @@ exports.run = function (workingPath, callback) {
       });
     },
     function (cb) {
+      const connection = createConnection();
```

Comment:
@jfromaniello thinking do I need to dispose this connection?

Reply:
I think it will be better if you use always the same client, since connections are costly and the client does polling automatically

Reply:
this is only created once on startup - for authentications the connection is reused...

Reply:
oh got it, then yes.. I think you should disconnect